src/zoid/auth/component.jsx

```diff
@@ -3,45 +3,54 @@
 /* eslint max-lines: 0 */
 
 import { node, dom } from 'jsx-pragmatic/src';
-import { getPayPalDomainRegex, getLogger, getLocale, getEnv, getClientID,
-    getSDKMeta, getCSPNonce, getBuyerCountry, getVersion, getPayPalDomain, getSessionID } from '@paypal/sdk-client/src';
+import {
+    getPayPalDomainRegex, getLogger,  getEnv, getClientID,
+    getSDKMeta, getCSPNonce,   getPayPalDomain, getSessionID
+} from '@paypal/sdk-client/src';
 import { create, CONTEXT, type ZoidComponent } from 'zoid/src';
 import { isDevice, supportsPopups, inlineMemoize } from 'belter/src';
-import { FUNDING } from '@paypal/sdk-constants/src';
 import { Overlay, SpinnerPage } from '@paypal/common-components/src';
 
-import type { AuthPropsType } from './props';
+// import {  validateResponseType  } from '../button/util';
+
+import { type AuthPropsType } from './props';
 import { DEFAULT_POPUP_SIZE } from './config';
 
+export type AuthComponent = ZoidComponent<AuthPropsType>;
+
 export function getAuthComponent() : ZoidComponent<AuthPropsType> {
     return inlineMemoize(getAuthComponent, () => {
+      
         const component = create({
             tag: 'paypal-auth',
             url: () => `${ getPayPalDomain() }${ __PAYPAL_IDENTITY__.__URI__.__AUTH__ }`,
-        
+
             attributes: {
                 iframe: {
                     scrolling: 'yes'
                 }
+
             },
-        
+
             defaultContext: supportsPopups() ? CONTEXT.POPUP : CONTEXT.IFRAME,
 
             domain: getPayPalDomainRegex(),
-        
+
             logger: getLogger(),
-        
+
             prerenderTemplate: ({ doc, props }) => {
                 return (
                     <SpinnerPage
-                        nonce={ props.nonce }
+                        nonce={ props.nonce } style
```

Comment:
Is `style` needed here? Seems like a typo
```suggestion
                        nonce={ props.nonce }
```

Reply:
done